Makefile

```diff
@@ -0,0 +1,33 @@
+DEV_DEP_LINE = $(shell cat Cargo.toml | grep -n "\[dev-dependencies\]" | cut -d : -f 1)
+LINE = $(shell echo $$(( $(DEV_DEP_LINE) - 2 )))
+LINES = $(shell cat Cargo.toml | wc -l)
+TAIL_LINES = $(shell echo $$(( $(LINES) - $(LINE) )))
+
+python:
+	@head -n $(LINE) Cargo.toml > cargo/$@/Cargo.toml
+	@echo "pyo3 = { version = \"0.18.0\", features = [\"abi3\", \"extension-module\"] }" >> cargo/$@/Cargo.toml
```

Comment:
This was pretty gross but I was in a rush.

Reply:
Would be nice to use a toml parser or something to edit the section and re-export.

Reply:
Or, template everything.

Reply:
yeah add an issue so we track it I think it's fine for now